src/vs/editor/common/config/editorOptions.ts

```diff
@@ -428,6 +428,7 @@ export interface IEditorOptions {
 	 */
 	suggest?: ISuggestOptions;
 	inlineSuggest?: IInlineSuggestOptions;
+	inlineEdit?: IInlineEditOptions;
```

Comment:
Maybe this should be called "experimentalInlineEdit", to make clear that this all might be removed/changed.

Reply:
✅